go/vt/mysqlctl/query.go

```diff
@@ -68,10 +68,18 @@ func (mysqld *Mysqld) ExecuteSuperQueryList(ctx context.Context, queryList []str
 	return mysqld.executeSuperQueryListConn(ctx, conn, queryList)
 }
 
+func limitString(s string, limit int) string {
+	if len(s) > limit {
+		return s[:limit]
+	}
+	return s
+}
+
 func (mysqld *Mysqld) executeSuperQueryListConn(ctx context.Context, conn *dbconnpool.PooledDBConnection, queryList []string) error {
 	for _, query := range queryList {
-		log.Infof("exec %v", redactMasterPassword(query))
+		log.Infof("exec %s", limitString(redactMasterPassword(query), 200))
```

Comment:
what does `200` stand for? Please consider making a global constant

Reply:
it is only for local use to limit logging of large queries generated while testing large json documents. Created a local const for improved readability.